src/mysql/create.sql

```diff
@@ -14,7 +14,9 @@ CREATE TABLE Major (
 
 CREATE TABLE User (
   id INT NOT NULL AUTO_INCREMENT,
-  name VARCHAR(255),
+  fname VARCHAR(255),
+  lname VARCHAR(255),
```

Comment:
using more descriptive names and snake_case to improve readability
```suggestion
  first_name VARCHAR(255),
  last_name VARCHAR(255),
```

Reply:
Done!